database.py

```diff
@@ -0,0 +1,100 @@
+from settings import NEO4J_USER, NEO4J_URI, NEO4J_PASSWORD
+from neo4j import GraphDatabase, exceptions
+from neo4j.exceptions import ServiceUnavailable
+import logging
+
+
+class DataBase:
+
+    def __init__(self):
+        self.driver = None
+        try:
+            self.driver = GraphDatabase.driver(uri=NEO4J_URI, auth=(NEO4J_USER, NEO4J_PASSWORD))
+        except Exception(exceptions.AuthError):
+            raise
+
+    def close(self):
+        if self.driver is not None:
```

Comment:
I would throw an error here as close should never be called off the driver is not there

Reply:
oh you are right I didn't think of that